props/saber_sabersense_buttons.h

```diff
@@ -465,6 +490,61 @@ class SabersenseButtons : public PROP_INHERIT_PREFIX PropBase {
     }
 #endif
   }
+  
+  // PRESET SKIPPING
+  // Skip Value definable.
+  // One Button System - Up, Horizontal, Down.
+  bool OneBtnMultiFontSkip(int skip_value) {
+      int count = 0;
+      CurrentPreset tmp;
+      for (int i = 0; ; i++) {
+          tmp.SetPreset(i);
+          if (tmp.preset_num != i) break;
+          count++;
+      }
+      float angle = fusor.angle1();
+      int new_index = current_preset_.preset_num;
+
+      if (angle < -M_PI / 6) { // Pointing down
+          new_index -= skip_value;
+      } else if (angle > M_PI / 6) { // Pointing up
+          new_index += skip_value;
+      } else { // Horizontal
+          SpeakBatteryLevel();
+          return true;
+      }
+      // Capping, not wrapping - less disorientating for user.
+      if (new_index < 0) new_index = 0;
+      if (new_index >= count) new_index = count - 1;
+      // Save state before setting the preset
+      SetPreset(new_index, true);
+#ifdef SAVE_PRESET
+      SaveState(new_index);
```

Comment:
SaveState is usually done before SetPreset to avoid glitches while playing font.wav